ui/templates/welcome.html

```diff
@@ -29,3 +30,14 @@
         </div>
     {% endif %}
 {% endblock %}
+{% block postcontent %}
+{{ block.super }}
+{% compress js %}
+  <script type="text/javascript"
```

Comment:
Thanks for investigating this, I can confirm this causes the problem to go away. However you should replace these lines with:

    <script type="text/javascript">
    require(["jquery", "bootstrap"]);
    </script>

On `repository.html` the problem doesn't appear because we require `listing.js` which in turn requires `jquery` and `bootstrap` among other things. It should be sufficient to require them on this page in the same way